zaza/charm_tests/security/tests.py

```diff
@@ -20,31 +20,20 @@
 
 import zaza.model as model
 import zaza.charm_lifecycle.utils as utils
+from zaza.utilities.file_assertions import (
+    assert_path_glob,
+    assert_single_file,
+)
 
 
-def _make_test_function(application, file_details):
+def _make_test_function(application, file_details, paths=[]):
```

Comment:
Switch default for path to None (https://docs.openstack.org/charm-guide/latest/coding-guidelines.html#avoid-dynamic-default-arguments-in-functions)